accountsdb-plugin-postgres/src/postgres_client.rs

```diff
@@ -66,6 +69,19 @@ pub struct DbAccountInfo {
     pub write_version: i64,
 }
 
+pub(crate) fn abort() -> ! {
+    #[cfg(not(test))]
+    {
+        // standard error is usually redirected to a log file, cry for help on standard output as
+        // well
+        println!("Validator process aborted. The validator log may contain further details");
```

Comment:
nit: Should probably use `eprintln!` here

Reply:
Done